semanticdb-java/src/main/protobuf/semanticdb.proto

```diff
@@ -103,12 +103,24 @@ message SymbolInformation {
   Kind kind = 3;
   int32 properties = 4;
   string display_name = 5;
+  repeated Annotation annotations = 13;
   Signature signature = 17;
   Access access = 18;
   repeated string overridden_symbols = 19;
   Documentation documentation = 20;
 }
 
+message Annotation {
+  Type tpe = 1;
+  // -- OUT OF SPEC -- //
+  message ParameterPair {
+    string key = 1;
+    Tree value = 2;
+  }
+  repeated ParameterPair parameters = 2;
```

Comment:
Curious why thats the case :thinking: could you explain? The spec doesnt currently have anything at `index 2`, so I would've thought this would be a non-issue to give `index 3`

Reply:
My bad, I though field 2 was already reserved in the spec.